.github/ISSUE_TEMPLATE/tracking_issue.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Tracking Issue
+about: A tracking issue for a feature in Rust.
+title: Tracking Issue for XXX
+labels: C-tracking-issue
+---
+<!--
+Thank you for creating a tracking issue! 📜 Tracking issues are for tracking a
+feature from implementation to stabilisation. Make sure to include the relevant
+RFC for the feature if it has one. Otherwise provide a short summary of the
+feature and link any relevant PRs or issues, and remove any sections that are
+not relevant to the feature.
```

Comment:
```suggestion
not relevant to the feature.

Remember to add team labels to the tracking issue.
For a language team feature, this would e.g., be `T-lang`.
Such a feature should also be labeled with e.g., `F-my_feature`.
This label is used to associate issues (e.g., bugs and design questions) to the feature.
```

Reply:
I don't think people can add labels unless they are a contributor, which is why I initially didn't include advice like this.

Reply:
Tracking issues like this are filed by contributors and team members.

Reply:
[They can add both `T-*` and `F-*` labels with triagebot.](https://github.com/rust-lang/rust/blob/ce361fb24f0896bf7d983549117cbe1f70f32dcf/triagebot.toml#L3-L7)

Reply:
@Centril Oh yes I was confused which template this one. Seems fine then.

@pietroalbini Does that require them to add `@rustbot T-lang` or can they use GitHub's UI?

Reply:
@XAMPPRocky I think that's `@rustbot modify labels: T-lang F-garbage_collection`